Nodejs/Product/Nodejs/Project/NodejsProjectNode.cs

```diff
@@ -161,10 +161,12 @@ internal override string GetItemType(string filename)
                 return node.ItemNode.ItemTypeName;
             }
 
+            // We need to return TypeScriptCompile for now to maintain backwards compatibility. In the future we will return "None" once the TypeScript SDK has been removed.
```

Comment:
>// We need to return TypeScriptCompile for now to maintain backwards compatibility. In the future we will return "None" once the TypeScript SDK has been removed. [](start = 12, length = 161)

Would it be worthwhile to file an issue for this so that you can locate all the places in need of updating by searching for the issue number?

Reply:
The issue most likely is going to be there for a long time since there is no plans on the short term to remove the backward compatibility. I prefer to track independently during the effort to remove the SDK.